docs/zello-sdk-documentation/com/zello/sdk/Zello.html

```diff
@@ -741,6 +753,30 @@ <h2 title="Class Zello" class="title">Class Zello</h2>
 </dl>
 </li>
 </ul>
+<a name="isLastMessageReplayAvailable--">
+<!--   -->
+</a>
+<ul class="blockList">
+<li class="blockList">
+<h4>isLastMessageReplayAvailable</h4>
+<pre>public&nbsp;boolean&nbsp;isLastMessageReplayAvailable()</pre>
+<div class="block">Determines if there is a message available to replay.
+ <p>
+     This method can be used to determine whether or not your UI should display a button to replay the last incoming message.
+           There are several places where you should call this method:
```

Comment:
Proposed revision: "The following two events may trigger a change in the last message availability." Otherwise it sounds like they actually **should** call this method.